pkg/apis/triggers/v1alpha1/cluster_trigger_binding_defaults.go

```diff
@@ -0,0 +1,24 @@
+/*
+Copyright 2019 The Tekton Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package v1alpha1
+
+import (
+	"context"
+)
+
+// SetDefaults initializes ClusterTriggerBinding ctb with its default values.
+func (ctb *ClusterTriggerBinding) SetDefaults(ctx context.Context) {}
```

Comment:
We should have a `var _ apis.Defaultable = (*ClusterTriggerBinding)(nil)` in there :angel:

Reply:
@vdemeester Thanks for the review.

I have added that now. I did not add because we nothing in default right now, and also the same was the case for TriggerBinding. But now I have added to both.